app/gui/src/ide/integration.rs

```diff
@@ -95,11 +95,13 @@ impl Model {
                     }
                     Err(err) => error!(logger, "Could not list projects: {err}."),
                 }
+            } else {
+                warning!(logger, "Project opening failed: no ProjectManagingAPI available.");
             }
         });
     }
 
-    /// Create a new project through project manager API.
+    /// Create a new project. `template` is an optional name of the template.
```

Comment:
```suggestion
    /// Create a new project. `template` is an optional name of the project template passed to the Engine.
```
The current sentence is not very informative :)

Reply:
Done